xcell/mappers/mapper_SDSS.py

```diff
@@ -94,7 +88,9 @@ def _get_signal_map(self):
 
     def get_signal_map(self):
         if self.signal_map is None:
-            fn = self._get_map_fname(mp='signal')
+            fn = '_'.join([f'SDSS_{self.SDSS_name}_signal',
```

Comment:
I kind of prefer to have fixed names for the mappers like here instead of relying on the user

Reply:
We only have this structure for SDSS and ACT. Doing this would be dangerous, though. For example, the `MapperBOSS` class can be either the CMASS or LOWZ samples, and those should have different names.

Moreover, imagine you want to divide them into different z-bins, so you could have `CMASS1`, `CMASS2` etc.

I think it's best to rely on the user.

Reply:
(which is us)